docker/linux/unittest/Dockerfile

```diff
@@ -16,6 +16,10 @@ FROM cobalt-base
 
 ARG HOME=/root
 
+RUN echo deb http://deb.debian.org/debian-debug \
+    $(. /etc/os-release && echo "$VERSION_CODENAME")-debug main \
```

Comment:
Env var references should be wrapped in curly braces:`${VERSION_CODENAME}`

Reply:
Amended